src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Car implements Comparable<Car> {
+    private final CarName name;
+    private final Position position
+            ;
```

Comment:
```suggestion
    private final Position position;
```
> 코드 자동 정렬하기 (Reformat Code)
MacOS: Cmd + Opt + l
Win/Linux: Ctrl + Alt + l

[[IntelliJ] intellij 유용한 단축키 정리](https://gmlwjd9405.github.io/2019/05/21/intellij-shortkey.html) 😉

Reply:
감사합니다 마지막에 꼭 한번 정리를 해야겠네요